src/data/value.rs

```diff
@@ -288,6 +288,7 @@ impl Value {
             (DataType::Boolean, AstValue::Boolean(v)) => {
                 Ok(nullable.into_value(Value::OptBool(Some(*v)), Value::Bool(*v)))
             }
+            (DataType::Text, AstValue::SingleQuotedString(v)) => Ok(Value::Str(v.to_string())),
```

Comment:
I noticed after the fact that others have solved this issue too.
Though both of them changed it to
`Ok(nullable.as_value(Value::OptStr(Some(v.clone())), Value::Str(v.clone())))`

Would that be better?

Reply:
Oh yes, that should be changed like you wrote :) thanks

Reply:
Done :-)